src/Spice86.Core/Emulator/Devices/Video/Registers/Register8.cs

```diff
@@ -0,0 +1,54 @@
+namespace Spice86.Core.Emulator.Devices.Video.Registers;
+
+using System.Runtime.CompilerServices;
+
+public class Register8 {
```

Comment:
Wondering if this could be mutualized with other devices we have. Probably not easily with the cpu registers as they are 32bits and provide access to 16 and 8 bits.

Reply:
I'm not sure what the benefits would be.